packages/extension-ui/src/Popup/ImportLedger.tsx

```diff
@@ -147,21 +147,24 @@ function ImportLedger ({ className }: Props): React.ReactElement {
       <VerticalSpace/>
       <ButtonArea>
         {ledgerLocked
-          ? (<Button
-            isBusy={ledgerLoading || isBusy}
-            onClick={refresh}
-          >
-            <FontAwesomeIcon icon={faSync} />
-            {t<string>('Refresh')}
-          </Button>
+          ? (
```

Comment:
Yes, I know it is more space-y ... beauty is in the eye of the beerholder I guess :)